python/orca/src/bigdl/orca/learn/pytorch/torch_runner.py

```diff
@@ -466,17 +466,22 @@ def get_state_dict(self):
 
     def load_state_dict(self, state):
         """Sets the state of the model."""
-        for model, state_dict in zip(self.models, state["models"]):
-            model.load_state_dict(state_dict)
+        import collections
+        if isinstance(state, collections.OrderedDict):
+            for model, state_dict in zip(self.models, [state]):
```

Comment:
Is this reasonable? It will always require self.models to be a singleton list.

Reply:
collections.singletonList? I think it's a java method and `[state]` should always be a python list type.

Reply:
No... I mean since [state] is of length 1, you do the zip, then you assume self.models is of length 1, does it add constraint/limitation to the current design?

Reply:
yes you're right, this should be updated.